rest-api/offline/metrics_pipeline.py

```diff
@@ -272,7 +272,7 @@ def map_answers(reader):
                              question_code != RACE_QUESTION_CODE):
       race_codes = [code_dao.get_code(PPI_SYSTEM, value) for value in race_code_values]
       race = get_race(race_codes)
-      if race == Race.SKIPPED:
+      if race == Race.PMI_Skip:
```

Comment:
I'm not sure you need this if statement anymore?

Reply:
removed, was gonna leave it because we explicitly cast PMI_Skip to string, wasn't 100% sure it wouldn''t effect something.